Docs 11618 cpu affinity v4.4 cv (#2129)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/starting-and-stopping-mule-esb.adoc

[NOTE]
====

> You can also start Mule without executing the installation step. This will start Mule in the background for the current user session.

Reviewer: ```suggestion
You can also start Mule without executing the installation step, which starts Mule in the background of the current user session.
```

Stopped Mule Enterprise Edition.
----

== See Also

> * xref:mule-runtime::runtime-installation-task.adoc[Download, Install, Configure, Upgrade Mule]

Reviewer: ```suggestion
* xref:mule-runtime::runtime-installation-task.adoc[Download, Install, Configure, and Upgrade Mule]
```
